modules/microshift-kickstart-prep.adoc

```diff
@@ -0,0 +1,67 @@
+//Do we use the Kickstart to provision a production-version RHEL for Edge also? "Kickstart scripts are plain text files that contain keywords recognized by the installation program, which serve as directions for the installation." from the RHEL docs; so is the installation program in our case image builder?
+
+.Prerequisites
+
+* The machine you are provisioning must meet the system requirements for installing {microshift-short}.
+* The file system must have a logical volume manager (LVM) volume group (VG) with sufficient capacity for the persistent volumes (PVs) of your workload.
+* A pull secret from the https://console.redhat.com/openshift/install/pull-secret[Red Hat Hybrid Cloud Console] must be present as `/etc/crio/openshift-pull-secret` and have root user-only read/write permissions.
+* The firewall must be configured with the required settings.
+//Should we add some guidance on the firewall settings/where to look for that info?
+
+.Procedure
+
+. Install the `microshift-release-info` RPM package containing the sample Kickstart files that are in the `/usr/share/microshift/kickstart` directory by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install -y microshift-release-info
+----
+//Q: is this right? is the kickstart in the release info rpm?
+
+. Install the utilities used during the Kickstart file creation by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install -y openssl gettext
+----
+
+. Set the variables pointing to secrets included in `kickstart.ks`.
+
+* The `PULL_SECRET` file contents are copied to the `/etc/crio/openshift-pull-secret` directory at the post-installation stage to authenticate {ocp} container registry access.
++
+.Example command setting the `PULL_SECRET` variable:
+[source,terminal]
+----
+$ export PULL_SECRET="$(cat ~/.pull-secret.json)" <1>
+----
+<1> Only the encrypted password is included in the Kickstart file.
+
+* The `USER_PASSWD` setting is used as an encrypted password for the `redhat` user for logging into the host.
+.Example command setting the `USER_PASSWD` variable.
+[source,terminal]
+----
+$ export USER_PASSWD="$(openssl passwd -6 "${PASSWD_TEXT}")"
+----
+
+* Set a password in the `PASSWD_TEXT` variable to use in the`USER_PASSWD` setting:
+.Example command setting the `PASSWD_TEXT` variable.
+[source,terminal]
+----
+$ PASSWD_TEXT=<my_redhat_user_plain_text_password> <1>
+----
+<1> Replace _<my_redhat_user_plain_text_password>_ with the password you want to use.
+
+. Follow the instructions for your installation type to create a working Kickstart file from a template.
+//So this is where we need to determine where to place all of the following content. We don't actually refer to image building anywhere else, so we have the two modules for RPM and bootc right here for editing purposes, but we need to figure out where they should really live.
```

Comment:
We have 3 ways of installing: bootc, ostree and RPM. Because the current section is a prereq for all the 3, the latter should be logically "under" the current section.